src/python/turicreate/test/test_sframe_s3.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2020 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+#
+from __future__ import print_function as _  # noqa
+from __future__ import division as _  # noqa
+from __future__ import absolute_import as _  # noqa
+from ..data_structures.sframe import SFrame
+from turicreate.util import _assert_sframe_equal
+
+import tempfile
+import os
+import shutil
+import pytest
+import boto3
+
+remote_sframe_folders = ["small_sframe_dc", "medium_sframe_ac", "big_sframe_od"]
+
+
+@pytest.mark.skipif(
+    os.environ.get("DTURI_ENABLE_SF_S3") is None,
```

Comment:
What's the reason for the D here?  TURI_ENABLE_SF_S3 would be more consistent.

Reply:
removed `D`